templates/app-settings.js

```diff
@@ -0,0 +1,8 @@
+AppSettings = {
```

Comment:
👍 
Let's add a reference to this on the README(?). Something like "How to add specific things to specific environments?" + "CORS issues in development"(with a link to Ionic Documentation about this)